packages/frontend/src/components/modules/documents/Faq/Faq.module.scss

```
@@ -0,0 +1,66 @@
+.title {
+  @include headline5;
+  position: relative;
+  margin: 59px 0 0 100px;
+
+  @include bp-1059 {
+    width: 688px;
+    padding-bottom: 24px;
+    margin: 0 auto;
+    margin-top: 92px;
```

А зачем отдельно прописываешь маржин-топ?
Почему не прописать все вместе?
```suggestion
    margin: 92px auto 0;
```